docs/Legal/Customer Agreement.mdx

```diff
@@ -0,0 +1,85 @@
+de mantener el Servicio funcionando según lo previsto.
+
+### Pago
+
+Si decide utilizar nuestros servicios, acepta pagar las tarifas asociadas con ello. Las tarifas se acuerdan entre ambas partes y pueden ser actualizadas de vez en cuando. Usted acepta pagar las tarifas de manera oportuna
+y proporcionarnos un método de pago válido. Si su método de pago falla repetidamente, nos reservamos el derecho de suspender o cancelar su acceso al Servicio, aunque primero se recopilará y enviará una copia de los datos almacenados
+a usted.
+
+### Propiedad Intelectual
+
+El nombre y el logotipo son propiedad intelectual de Harmony por derechos de autor. Cualquier uso indebido de estos recursos será perseguido.
+
+### Terminación
+
+Podemos terminar o suspender su acceso al Servicio de inmediato, con previo aviso y enviando una copia de los datos de sus beneficiarios resultantes de un incumplimiento de este Acuerdo.
+
+### Limitación de Responsabilidad
+
+Harmony no será responsable de ningún daño indirecto, incidental, especial, consecuencial o punitivo que resulte de su uso o incapacidad para usar el Servicio.
+
+## Política de Privacidad
+
+### Recopilación de Información
+
+Recopilamos datos necesarios para que el sistema funcione, incluidas, entre otras, las cuentas de usuario y las preferencias. Todos los datos sensibles serán encriptados y accesibles únicamente por el cliente.
+
+### Uso de la Información
+
+Utilizamos la información recopilada únicamente para proporcionar y mejorar nuestro servicio en la nube. No vendemos ni compartimos la información del usuario con terceros sin su consentimiento.
+
+### Seguridad de los Datos
+
+Tomamos medidas razonables para proteger la seguridad de la información del usuario, sin embargo, cualquier transmisión de datos a través de Internet no es completamente segura y no podemos garantizar ni 
+ser responsables de la seguridad de la información transmitida hacia o desde el servicio en la nube en caso de que el dispositivo del usuario final sea comprometido.
+
+### Retención de Datos
+
+Retenemos la información del usuario solo durante el tiempo necesario para cumplir con los propósitos descritos en esta Política de Privacidad. Para obtener más información, consulte nuestra 
+[Política de Protección de Datos](Data%20Protection.mdx).
+
+### Servicios de Terceros
+
+Nuestro servicio en la nube puede integrarse con servicios de terceros, y su uso de la información está regido por sus respectivas políticas de privacidad.
+
+## Cambios en la Política de Privacidad
+
+Nos reservamos el derecho de actualizar o cambiar nuestra Política de Privacidad en cualquier momento. Cualquier cambio será efectivo inmediatamente después de publicar la Política de Privacidad revisada en nuestro sitio web, momento en el cual se notificará al usuario
```

Comment:
I recommend that this sentence be re-structured to improve customer confidence. "In the event of a change to the Privacy Policy, a communication will be made prior to the modification of the policy so that users can review the changes made. After a period of 5 days the policy will be published on our website and the user will be notified again".

Reply:
done